<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>59</y>
      <width>991</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="front">
     <attribute name="title">
      <string>front</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="back">
     <attribute name="title">
      <string>back</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>210</y>
        <width>571</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Wind speed</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Wind direction</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Desired course</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>True airspeed</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Wind correction angle</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>True ground speed</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="degrees">
         <property name="text">
          <string>degrees</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="radians">
         <property name="text">
          <string>radians</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="wind_speed">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="wind_direction">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="desired_course">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="true_airspeed">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QTextEdit" name="wind_correction_angle">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QTextEdit" name="true_ground_speed">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
